angular.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
   "version": 1,
-  "newProjectRoot": "projects",
+  "newProjectRoot": "feature-libs",
```

Comment:
This is to inform `ng cli` where to create new libs?

Reply:
Yes, correct. As we are moving away from `projects`, I think `feature-libs` should be the new project roon